app/views/pages/catalogo/section_1.26.html

```diff
@@ -873,6 +873,12 @@ <h1>Autori di fisonomia</h1>
                </li>
             </ol>
          </section>
+         <a rel="prev"
+            class="btn btn-default btn-sm"
+            href="/catalogo/section_1.25">« Previous</a>
+         <a rel="next"
+            class="btn btn-default btn-sm"
+            href="/catalogo/section_2.0">Next »</a>
```

Comment:
Notice how it handles going to the next section (2.0) at the end of this section (1.26)